Directory.Build.targets

```diff
@@ -36,13 +37,18 @@
 
   <Import Project="Sdk.targets" Sdk="Microsoft.DotNet.Arcade.Sdk" />
 
+  <PropertyGroup>
+    <FullNugetVersion>$(VersionPrefix)-$(PreReleaseVersionLabel)</FullNugetVersion>
```

Comment:
Any idea why this was a target before, rather than a property group? It's interesting that the approach of re-calculating the version number was taken. Typically a repo will build a target in a project it knows is non-stable and pull the version number as an output property.

Reply:
This commit explains it: https://github.com/dotnet/installer/commit/83679698f989f763a7d621dfb01ce35d16a72950

But this wasn't necessary even back then as all that information is available at evaluation time in a Directory.Build.targets file AFTER the Arcade's Sdk.targets is imported.